ci/ruby.yml

```diff
@@ -17,6 +17,9 @@ jobs:
   test:
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.6', '2.7', '3.0']
```

Comment:
Adding a matrix seems good to me and aligns with our other CI workflows (Python, Node).